packages/www/layouts/dashboard.tsx

```diff
@@ -77,7 +77,7 @@ function DashboardLayout({
       url,
       images: [
         {
-          url: image ? image.url : "https://livepeer.studio/img/OG.png",
+          url: image ? image.url : "https://assets.livepeer.studio/api/og",
```

Comment:
Hah I've made this suggestion a few times, but the image here also includes the word "onchain" and not sure where that's defined.